Documentation/markdown/SetupHelp.md

```diff
@@ -1,126 +1,147 @@
+3. **Qt 5.12.1** is installed and included in PATH (environment variable) - find [here](https://www.qt.io/download-open-source).
 
 	*	Download Qt Online Installer 
-	*	In the meantime, your computer is downloading Qt installer, create Qt account. 
-	*	Sign in with your new account to Qt installer and select directory, where Qt will be installed. (`C:\Qt` should be default option, if not write **this** path out)
-	*	Select components, you want to install:
+	*	While your computer is downloading Qt installer, create Qt account. 
+	*	Sign in with your new account to Qt installer and select directory, where Qt will be installed (`C:\Qt` should be default option).
+	*	Select components to install:
 
 		* Check the *Archive* box
-		* Select *Filter*
+		* Click *Filter*
 		* Open section **Qt 5.12.1**
 		* **Mandatory:** Select *binaries x64 msvc2017*
 		* **Mandatory:** Select *mingw*
 		
 		![](./fig/Qt_Installation_settings.png)
 
-		* **HINT:** There are components, which Qt Online Installer selects as default options. You can uncheck these components for saving computer memory.
+		* *HINT:* There are components, which Qt Online Installer selects as default options. You can uncheck these components for saving computer memory.
 
-4. **Boost 1_65_1** is installed (find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)).
+4. **Boost 1_65_1** is installed - find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download).
 
-	* Create a folder named thirdparty in `C:\` and install **Boost 1_65_1** to `C:\thirdparty\vs2015\x64\boost_1_65_1`.
-	* Add this path into the environment variables. (Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2015\x64\boost_1_65_1`).
+	* Create a folder named thirdparty in `C:\` and install **Boost 1_65_1** to `C:\thirdparty\vs2017\x64\boost_1_65_1`.
+	* Add this path to the environment variables. (Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2017\x64\boost_1_65_1`).
 
-5. **Anaconda 2** (version with Python 2.7)  - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe) (For Windows users) 
+5. **Anaconda 2** (version with Python 2.7) - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe). 
 
 
-***
 ## <a name="Setup"></a> Setup
 
 ### <a name="Source_code"></a> Download source code 
 
- Clone **OpenInfraPlatform** repository  - more information you can find [here](./GitProcess.md)
+ Fork & clone [Open-Infra-Platform repository](https://www.github.com/tumcms/Open-Infra-Platform) - for detailed instuctions consult our [Git Guideline](./GitProcess.md).
 
 ### <a name="Prep_solution"></a> Preparing solution 
```

Comment:
In Line 12 you have replaced **Preparing solution** with **Generating solution**, so I guess it should be also replaced here with **Generating solution**.